src/test/java/org/gridsuite/study/server/StudyControllerDynamicSimulationTest.java

```diff
@@ -671,14 +610,14 @@ public List<MappingInfos> answer(InvocationOnMock invocation) {
                 .andExpect(status().isOk()).andReturn();
         String content = result.getResponse().getContentAsString();
 
-        assertFalse("Content not null or empty", Strings.isBlank(content));
+        assertThat(Strings.isBlank(content)).as("Content not null or empty").isFalse();
```

Comment:
```suggestion
        assertThat(content).isNotBlank();
```

Reply:
Agreed. No need to keep the specific error message. The default error message is clear enough : `Expecting not blank but was: ""`